pyproject.toml

```
@@ -1,7 +1,76 @@
 [build-system]
-requires = ["setuptools>=42"]
+requires = ["setuptools>=61.2.0", "wheel"]
```

```suggestion
requires = ["setuptools>=61.2.0"]
```

You don’t need wheel - it’s added by setuptools via PEP 517 and it’s likely to be going away at some point.